src/plugins/vis_type_timeseries/public/application/components/vis_editor.js

```diff
@@ -219,6 +227,10 @@ export class VisEditor extends Component {
 
   componentDidMount() {
     this.props.renderComplete();
+
+    if (this.props.isEditorMode && this.props.eventEmitter) {
```

Comment:
nit: it's not related to that PR, but I see one unused method in that file `isValidKueryQuery`. Maybe if you already  touched that file can we remove that too?

Reply:
Removed